packages/salesforcedx-vscode-core/src/messages/i18n.ts

```diff
@@ -128,5 +128,9 @@ export const messages = {
   isv_debug_bootstrap_step1_create_project:
     'SFDX: ISV Debugger Setup, Step 1 of 2: Create Project',
   isv_debug_bootstrap_step1_configure_project:
-    'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project'
+    'SFDX: ISV Debugger Setup, Step 2 of 2: Configure Project',
+  force_apex_log_fetch_text: 'SFDX: Fetch Apex Debug Logs...',
+  force_apex_log_fetch_no_logs_text: 'No Apex debug logs were found',
+  force_apex_log_fetch_pick_log_text: 'Pick an Apex debug log to fetch',
+  force_apex_log_list_text: 'Fetching Apex debug logs'
```

Comment:
@ruthemmanuelle please take a look at these messages

Reply:
So all these should be changed from `fetch` to `get`.